integrations/event-handler/events_job.go

```diff
@@ -107,6 +112,10 @@ func (j *EventsJob) runPolling(ctx context.Context) error {
 				return trace.Wrap(err)
 			}
 
+			eventsProcessed++
+		case <-logTicker.C:
+			log.WithField("eventsPerMinute", eventsProcessed).Info("event processing rate")
```

Comment:
Can we integrate Prometheus metrics into it? It would likely be worth the effort as it would provide us with better metrics.

Additionally, we need to determine the source of the slowness. Is it due to Teleport taking an excessive amount of time to read data from Athena (in which case, we might need to modify the stream API), or is it because our highly inefficient single worker is making a POST request for each event? 😢

Reply:
This PR is just about landing a short-term improvement ASAP.  I agree that we need better monitoring overall, but thats something that should be tackled elsewhere.

My testing is ongoing, but I can say that in my test cases ingesting events from Athena is consistently *much* slower than exporting the event downstream.  Right now it's looking like Athena is definitely the bottleneck.